environment-api/src/main/java/com/sequenceiq/environment/api/v1/proxy/model/response/ProxyResponse.java

```diff
@@ -56,4 +56,13 @@ public String getCreator() {
     public void setCreator(String creator) {
         this.creator = creator;
     }
+
+    @Override
+    public String toString() {
+        return super.toString() + ", " + "ProxyResponse{" +
+                "crn='" + crn + '\'' +
+                ", userName=" + userName +
```

Comment:
This is a `SecretResponse`. While it does not reveal anything sensitive, I wonder if it is worth including this in the output.